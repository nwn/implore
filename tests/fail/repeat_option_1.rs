use implore::impl_op;

#[derive(Copy, Clone, PartialEq, Eq, Debug)]
struct X;

#[impl_op(autoref, autoref)]
fn neg(lhs: &X) -> X {
    *lhs
}

fn main() {}
