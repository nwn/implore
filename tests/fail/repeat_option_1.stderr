error: repeated option: `autoref`
 --> $DIR/repeat_option_1.rs:6:20
  |
6 | #[impl_op(autoref, autoref)]
  |                    ^^^^^^^
  |
note: previously occurs here
 --> $DIR/repeat_option_1.rs:6:11
  |
6 | #[impl_op(autoref, autoref)]
  |           ^^^^^^^
